tweet-impl-mastodon4j/build.gradle

```diff
@@ -25,7 +25,6 @@
 dependencies {
     api project(':tweetwallfx-tweet-api')
 
-    implementation 'org.json:json:20240303' // explicit declaration to avoid CVE-2022-45688
```

Comment:
`org.mastodon4j:mastodon4j-core:0.10.0` has transient `org.json:json:20240205` which does not have the listed CVE. We (currently) no longer need to upgrade the dependency.